.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+name: Add Tags to Release
+
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+jobs:
+  build:
```

Comment:
I'd be interested in another name for this. We are already using `build` in the ci-cdy.yml. Maybe build-release, or release

Reply:
release is fine with me.  thought it was a keyword so i just reused.  i will make the change.